### ISSUE TITLE
Create landing page

### ISSUE THREAD

**Issue by jukent** [ProjectPythiaTutorials/tutorial-template#6]
We want a landing page that we can generate a site from that will contain a table with every tutorial repository in this organization (save the template) and a link to that repository's Binder.

Perhaps we want this in a new repository within the ProjectPythiaTutorials organization, and we want to move the current`README.md` from #1 there as well (and replace it with the `README_template.md`). Thoughts?

**Comment by jukent** [comment]
Should `contributing.md` move to this new repository as well?

**Comment by jukent** [comment]
@kmpaul You mentioned this at the end of our meeting on Monday. Could you elaborate more on your thoughts here?

**Comment by kmpaul** [comment]
Yes.  There should be a repo in this new organization that has the GitHub Pages content, and then we should link that GitHub Page to something like `tutorials.projectpythia.org` so it looks like a cohesive unit.  Basically, that means we need a repository in this new organization called something like `projectpythiatutorials.github.io` (like the portal is in `projectpythia.github.io`), and then that repository will host a Jupyter Book like the Pythia Foundations book.

**Comment by kmpaul** [comment]
However, I don't think the `*.github.io` format for repos is actually required any more.  So, it could be called anything.

**Comment by jukent** [comment]
I'll start with that, and we can edit if we decide to later.
